l10n_it_withholding_tax/models/account.py

```diff
@@ -75,7 +75,13 @@ def create(self, vals):
                 )
                 == 1
             ):
-                vals.update({"amount": invoice.amount_net_pay})
+                vals.update(
+                    {
+                        "amount": invoice.amount_net_pay,
+                        "credit_amount_currency": invoice.amount_net_pay,
```

Comment:
Non so se ci sia la casistica di una fattura emessa in valuta con ritenuta d'acconto, sarebbe da verificare se la valuta della fattura è uguale alla valuta aziendale e in caso contrario convertire no?